recipes/parsimonious/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "parsimonious" %}
+{% set version = "0.8.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "ae0869d72a6e57703f24313a5f5748e73ebff836e6fe8b3ddf34ea0dc00d086b" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - python==2.7|>=3.4
```

Comment:
Please don't include these Python requirement.  These are satisfied implicitly because conda-forge only builds by deafult on 2.7, 3.6, and 3.7

Reply:
good to know, thanks!